src/contexts/LoginContext.tsx

```diff
@@ -1,41 +1,67 @@
 import axios from "axios"
 import { isEmpty, omitBy } from "lodash"
-import { createContext, useEffect, useContext } from "react"
+import {
+  createContext,
+  useEffect,
+  useContext,
+  PropsWithChildren,
+  useCallback,
+} from "react"
 
 import { LOCAL_STORAGE_KEYS } from "constants/localStorage"
 
 import { useLocalStorage } from "hooks/useLocalStorage"
 
 const { REACT_APP_BACKEND_URL: BACKEND_URL } = process.env
 
-const LoginContext = createContext(null)
+interface LoginContextProps {
+  userId: string
+  email: string
+  contactNumber: string
+  logout: () => Promise<void>
+  verifyLoginAndSetLocalStorage: () => Promise<void>
+}
+
+interface loggedInUserProps {
```

Comment:
types should be UpperCamelCased; we should also rename this to `LoggedInUser` as this isn't provided as an argument anywhere (we can also update `LoginContextProps` to extend this as well)

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/959/commits/b4234e06da950f69b585897da54f414caddd7b74